pkgs/tools/misc/nvimpager/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, pkgs
+# Defaults to false until I find out how to provide /dev/tty inside the
+# sandbox.
+, doCheck ? false
+}:
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "nvimpager";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "lucc";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1xy5387szfw0bp8dr7d4z33wd4xva7q219rvz8gc0vvv1vsy73va";
+  };
+
+  buildInputs = with pkgs; [
+    ncurses # for tput
+    neovim
+    procps # for nvim_get_proc() which uses ps(1)
+    pandoc
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+  buildFlags = [ "nvimpager.configured" ];
+  preBuild = ''
+    substituteInPlace nvimpager \
+      --replace '/usr/bin/env bash' '${pkgs.bash}/bin/bash' \
```

Comment:
This should be done by either invoking `patchShebangs` or by setting `/usr/bin/env bash` to `${pkgs.runtimeShell}`.